app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -114,12 +129,29 @@
       window.GOVUK.setDefaultConsentCookie()
     }
 
-    if (COOKIE_CATEGORIES[cookieName]) {
-      var consent = JSON.parse(window.GOVUK.getCookie('cookie_policy'))
+    currentConsentCookie = window.GOVUK.getConsentCookie()
 
+    // Survey cookies are tested for in Static, which we need to allow
```

Comment:
I think this comment is misleading - it sounds like this code is only for tests, which isn't true. I would just say that survey cookies behave differently (wildcard behaviour / dynamically generated) so we look for those separately.